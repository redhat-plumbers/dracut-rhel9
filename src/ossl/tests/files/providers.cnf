openssl_conf = openssl_init

[openssl_init]
providers = providers_sect

[providers_sect]
default = default_sect
fips = fips_sect
legacy = legacy_sect
base = base_sect
pkcs11 = pkcs11_sect
oqs = oqs_sect

[default_sect]
activate = 1

[fips_sect]
activate = 1

[legacy_sect]
activate = 1

[base_sect]
activate = 1

[pkcs11_sect]
activate = 1

[oqs_sect]
activate = 1
module = /usr/lib64/ossl-modules/oqsprovider.so.0.6.0
