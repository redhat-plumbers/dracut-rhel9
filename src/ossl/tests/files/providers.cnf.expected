/usr/lib64/ossl-modules/fips.so
/usr/lib64/ossl-modules/legacy.so
/usr/lib64/ossl-modules/pkcs11.so
/usr/lib64/ossl-modules/oqsprovider.so.0.6.0
